SparkPost/2web2ui
# FE-1346 Replace 'master account' with 'primary account'

### What Changed
- Replaces `master account` strings with `primary account`.
- 
### How To Test
- Run and verify unit and integration tests
- Verify the following:
  -  The `Subaccount` label component
  -  The `SubaccountSection` component
  - Can be verified via the following:
    - AB test create subaccount field help text
    - Suppression create subaccount field help text
    - Suppression upload subaccount field help text
    - Suppression table entry assigned to primary account
    - Domains create page subaccount radio group
    - Data privacy page subaccount options
    - Report builder & summary report group by table with a primary account row
    - Signals (HS, Spam Traps, Eng) table cells with a primary account row
    - Signals (HS, Spam Traps, Eng) table subaccount filter
    - Webhooks create subaccount radio group
    - Webhook entry assigned to primary account and primary and all

### To Do
